### PR TITLE
kafka 0.11

### PR DESCRIPTION
**Pull request by stevenschlansker**

### CODE REVIEW

File: src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java [hunk 1]
```diff
@@ -117,6 +117,7 @@ private KafkaConfig createConfig()
         config.put(KafkaConfig.DeleteTopicEnableProp(), "false");
         config.put(KafkaConfig.ControlledShutdownRetryBackoffMsProp(), "100");
         config.put(KafkaConfig.LogCleanerDedupeBufferSizeProp(), "2097152");
+        config.put(KafkaConfig.OffsetsTopicReplicationFactorProp(), (short) 1);
```
Comment (pennello) [inline src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java]: Cool.

Comment (pennello) [inline src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java]: I haven't reviewed the provisions of this library in detail, but is there another place where we can conveniently set this replication factor to a non-1 value by default?  I recall a problem where it defaulted to 1, so we didn't have any redundancy (e.g., some of the chat2 topics on chimera kafka).

Comment (stevenschlansker) [inline src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java]: Default values are set in the broker configuration, and apps can override them if need be, but I don't think there's a global "right" value.

Comment (pennello) [inline src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java]: 2, or maybe even 3, are much better than 1.  1 is almost never going to be what the user wants.  We should add some documentation of this too so users of this library won't be as surprised.  Isn't it a pain to change replication after topic creation?

**Pull request merged**